dev/local/Makefile

```
@@ -37,11 +37,15 @@
 #   test=         Only runs tests in the directories provided here, e.g.
 #                 repos/delphi/delphi-epidata/tests/acquisition/covidcast
 #   sql=          Overrides the default SQL connection string.
-
+#   m1=1          Mandatory if your local system uses Apple M1 chip
 
 # Set optional argument defaults
+ifdef m1
+	override m1 =--platform linux/amd64
+endif
+
 ifdef pdb
-	override pdb=--pdb
+	override =--pdb
```

issue: variable name removed from assignment, likely by accident

```suggestion
	override pdb=--pdb
```